plugins/transform/README.md

```diff
@@ -0,0 +1,78 @@
+# Flatfile Transform Plugin
+
+This plugin provides utilities for transforming and validating records in Flatfile.
+
+## Methods
+
+### compute
+
+Transforms the value of a field based solely on the intial value of that field. Optionally pass a message to surface to the user about the transformation.
+
+```ts
+client.use(
+  recordHook('my-sheet', (record: FlatfileRecord) => {
+    compute(
+      record,
+      'firstName',
+      (value) => value && value.toString().toLowerCase(),
+      // optional
+      'First name was changed to lower case.'
+    )
+    return record
+  })
+)
+```
+
+### transform
+
+Transforms the value of a field based on other fields in the record.
+
+```ts
+client.use(
+  recordHook('my-sheet', (record: FlatfileRecord) => {
+    transform(
+      record,
+      'fullName',
+      (record) => `${record.get('firstName')} ${record.get('lastName')}`,
```

Comment:
🤔   Could we do `fullName` as the param rather than record?

Reply:
As we discussed offline, the above `compute` method does this. Open to naming / syntax that makes this more clear.